README.md

```diff
@@ -12,12 +13,23 @@ REST API for managing cache control information for pages within the legacy CMS
 
 ### Configuration
 
-| Environment variable         | Default | Description                                                                                                        |
-|------------------------------|---------|--------------------------------------------------------------------------------------------------------------------|
-| BIND_ADDR                    | :29100  | The host and port to bind to                                                                                       |
-| GRACEFUL_SHUTDOWN_TIMEOUT    | 5s      | The graceful shutdown timeout in seconds (`time.Duration` format)                                                  |
-| HEALTHCHECK_INTERVAL         | 30s     | Time between self-healthchecks (`time.Duration` format)                                                            |
-| HEALTHCHECK_CRITICAL_TIMEOUT | 90s     | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format) |
+| Environment variable               | Default                                                                                                                                                                                                   | Description                                                                                                         |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |                
```

Comment:
Small one, but this table is not formatted correctly now. For instance, this line has some extra spaces at the end, and there are some spaces between `|` and `-`. Your IDE should hopefully help you to format this automatically.